client/my-sites/checkout/src/components/item-variation-picker/util.ts

```diff
@@ -23,12 +24,11 @@ export function getItemVariantCompareToPrice(
 		return compareTo.priceBeforeDiscounts * 2;
 	}
 
-	// CompareTo price with introductory offers (For Jetpack)
+	// CompareTo price with introductory offers and wihtout (For Jetpack and Akismet)
 	if (
-		compareTo.introductoryInterval === 1 &&
-		compareTo.introductoryTerm === 'year' &&
-		variant.introductoryInterval === 2 &&
-		variant.introductoryTerm === 'year'
+		isJetpackPlan( { product_slug: compareTo.productSlug } ) ||
+		isJetpackProduct( { product_slug: compareTo.productSlug } ) ||
+		isAkismetProduct( { product_slug: compareTo.productSlug } )
```

Comment:
Regardless of whether there is an introductory offer or not, the compareTo price is calculated the same for Jetpack and Akismet products.

Reply:
Does this change still allow for WPCOM introductory offers to correctly display the upsell pricing changes?

Reply:
Actually, no it doesn't.  My initial code change is still needed for Jetpack and Akismet products to properly show percentages for discounted prices (non-introductory-offer discounts), but I'm realizing the previous code block should still remain in place so that introductory-offers display correctly for products that are Not Jetpack or Akismet (such as WPCOM products). 
So I'll create another small PR and add the previous code block back in there. 👍

Reply:
Created an improvement PR here: https://github.com/Automattic/wp-calypso/pull/84447